digits/job.py

```diff
@@ -22,6 +22,7 @@ class Job(StatusCls):
     Base class
     """
     SAVE_FILE = 'status.pickle'
+    NON_PERSISTENT_JOB_DELETE_TIMEOUT_SECONDS = 3600
```

Comment:
that makes it a class attribute, is this what you want? or shall this be an instance attribute?